task/02-numbers-tasks.js

```diff
@@ -143,7 +143,7 @@ function getLastDigit(value) {
  * '-525.5'     => -525.5
  */
 function parseNumberFromString(value) {
-    throw new Error('Not implemented');
+    return value*1;
```

Comment:
not a clear solution
see https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Number